src/_scss/pages/interactiveDataSources/index.scss

```diff
@@ -0,0 +1,198 @@
+.usa-da-interactive-data-sources-page {
+  @import "all";
+  @import "layouts/default/default";
+  @import "components/pageLoading";
+  @import "layouts/default/stickyHeader/header";
+  @import "mixins/profilePage";
+
+  @include display(flex);
+  @include justify-content(flex-start);
+  @include align-items(center);
+  @include flex-direction(column);
+
+  min-height: 100vh;
+  width: 100%;
+
+  .site-header,
+  .usda-page__container,
+  .main-content,
+  .footer-container {
+    width: 100%;
+    .usda-page-header__toolbar {
+      .usda-download-btn .usda-button {
+        padding: 0;
+        margin: 0;
+      }
+      .usda-download-btn span,
+      .usda-share-icon span {
+        margin-top: rem(5);
+      }
+    }
+  }
+
+  .accordion-container {
+    @media (min-width: $large-screen) {
+      margin-left: rem(10);
+    }
+  }
+
+  .accordion-container .heading {
+    height: 14px;
+    color: $color-primary;
+    border: none;
+    font-size: $font-size-14;
+    max-width: rem(120);
+    padding: 0;
+    margin: 0;
+    justify-content: unset !important;
+
+    &:hover,
+    &:active,
+    &:focus,
+    &:visited {
+      color: $color-primary-darker;
+    }
+
+    @media (min-width: $large-screen) {
+      padding-top: rem(8);
+    }
+  }
+
+  .accordion-container .plus {
+    width: 10px;
+    height: 10px;
+    &:hover,
+    &:active,
+    &:focus,
+    &:visited {
+      color: $color-primary-darker;
+    }
+  }
+
+  .accordion-container .minus {
+    width: 10px;
+    height: 10px;
+    &:hover,
+    &:active,
+    &:focus,
+    &:visited {
+      color: $color-primary-darker;
+    }
+  }
+  .accordion-container .toggle {
+    color: $color-primary;
+    font-size: $font-size-14;
+    font-weight: $font-semibold;
+    padding-left: rem(6);
+    &:hover,
+    &:active,
+    &:focus,
+    &:visited {
+      color: $color-primary-darker;
+    }
+    @media (min-width: $large-screen) {
+      padding-right: rem(6);
+    }
+  }
+
+  .accordion-container .open .content {
+    display: none;
+  }
+
+  .usda-page-header__toolbar
```

Comment:
Jeez this is a super specific selector. Don't have a great feel for the codebase for if this is common or not, but definitely strikes me as something to be avoided if possible

Reply:
This is common in the codebase as it was found in 7 other files outside of the interactiveDataSources folder.